packages/js/create-woo-extension/src/index.js.mustache

```diff
@@ -3,14 +3,63 @@
  */
 import { addFilter } from '@wordpress/hooks';
 import { __ } from '@wordpress/i18n';
+import { Dropdown } from '@wordpress/components';
+import * as Woo from '@woocommerce/components';
+import { Fragment } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import './index.scss';
 
 const MyExamplePage = () => (
-	<h2>{ __( 'My Example Extension', '{{textdomain}}' ) }</h2>
+	<Fragment>
+		<Woo.Section component="article">
+			<Woo.SectionHeader title={ __( 'Search', 'my-extension-name' ) } />
```

Comment:
```suggestion
			<Woo.SectionHeader title={ __( 'Search', '{{textdomain}}' ) } />
```

In case somebody leaves this content behind and starts building around it, do we want to use the variable provided to mustache for the text domain here and in the other instances below?